linker_scripts/object_script.txt

```diff
@@ -152,6 +152,8 @@ D_06000228 = 0x06000228;
 D_06000A20 = 0x06000A20;
 D_06000040 = 0x06000040;
 
+/* z_bg_goron_oyu */
+D_6000988 = 0x6000988;
```

Comment:
```suggestion
/* z_bg_goron_oyu */
D_06000988 = 0x06000988;
```
Including the extra 0 seems more consistent with what's already here

Reply:
Also the object script is now ordered alphabetical, so would be nice if you could move it to the correct place.